projects/storefrontlib/src/lib/product/components/product-details/product-summary/product-summary.component.html

```diff
@@ -1,24 +1,42 @@
-<div class="cx-product-summary">
-  <div class="cx-product-summary__price cx-product-summary__price" aria-label="new item price">
-    {{product?.price?.formattedValue}}
-  </div>
-
-  <div class="cx-product-summary__description">
-    <p [innerHTML]="product?.summary"> </p>
-  </div>
+<div class="product-summary">
+  <ng-container *cxOutlet="outlets.TITLE">
+    <div class="item-title">
+      <div class="name">{{ product?.name }}</div>
+      <div class="code">ID {{ product?.code }}</div>
+    </div>
+  </ng-container>
 
-  <div class="cx-product-summary__variant">
-    <div class="cx-product-summary__variant-label">
-      Qty:
+  <ng-container *cxOutlet="outlets.RATING">
+    <div class="item-rating">
+      <cx-star-rating [rating]="product?.averageRating" [disabled]="true"></cx-star-rating>
+      <div class="review-count">({{ product?.numberOfReviews }})</div>
+      <a class="review-link btn-link">Write a review</a>
     </div>
-    <cx-item-counter [min]="1" [max]="product.stock.stockLevel" *ngIf="product?.stock?.stockLevel > 0 || product?.stock?.stockLevelStatus === 'inStock'"
-      (update)="updateCount($event)"></cx-item-counter>
-    <span class="cx-product-summary__variant-info">
-      <span *ngIf="product?.stock?.stockLevel > 0 || product?.stock?.stockLevelStatus === 'inStock'">
-        {{product?.stock?.stockLevel}} in stock</span>
-      <span *ngIf="product?.stock?.stockLevel == 0">Out of stock</span>
-    </span>
-  </div>
+  </ng-container>
 
+  <div class="price" aria-label="new item price">
+    {{ product?.price?.formattedValue }}
+  </div>
+  <div class="description"><p [innerHTML]="product?.summary"></p></div>
 
-</div>
\ No newline at end of file
+  <ng-container *cxOutlet="outlets.ADDTOCART">
+    <div class="quantity">
+      <div class="quantity-label">Qty:</div>
+      <div class="quantity-counter">
+        <cx-item-counter [min]="1" [max]="product.stock.stockLevel" *ngIf="product?.stock?.stockLevel > 0 || product?.stock?.stockLevelStatus === 'inStock'"
+          (update)="updateCount($event)"
+        ></cx-item-counter>
+        <span class="quantity-info">
+          <span *ngIf="product?.stock?.stockLevel > 0 || product?.stock?.stockLevelStatus === 'inStock'">
+            {{ product?.stock?.stockLevel }} in stock</span>
+          <span *ngIf="product?.stock?.stockLevel == 0">Out of stock</span>
+        </span>
+      </div>
+      <cx-add-to-cart
```

Comment:
Can we move the add-to-cart component outside the quantity element? Then we can rearrange the summary more easier (i.e. move description below add-to-cart, move add-to-cart before counter, etc).